internal/clients/ec2/aws_supported_types.go

```diff
@@ -0,0 +1,45 @@
+package ec2
+
+import (
+	"context"
+
+	"github.com/RHEnVision/provisioning-backend/internal/clients"
+	"github.com/RHEnVision/provisioning-backend/internal/payloads"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+)
+
+func FilterAWSInstanceTypes(_ context.Context, it *[]clients.InstanceType) {
+	for _, i := range *it {
+		if IsSupported(i) {
+			// If the Instance type is present in the yml it is supported
+			i.Supported = true
+		}
+	}
+}
```

Comment:
Kind of don't like the function name, it's not filtering but it's ok.

But why this is exported? I would expect this to be called from `NewInstanceType` and nowhere else.

Reply:
I removed that function as Ondrej suggested https://github.com/RHEnVision/provisioning-backend/pull/169#discussion_r954642225